util/util.go

```diff
@@ -703,7 +703,7 @@ func RemountVolume(volumeName string) error {
 	fsType, err := DetectFileSystem(volumeName)
 	if err != nil {
 		logrus.Warnf("Failed to detect the filesystem for volume %v, will skip remounting it: %v", volumeName, err)
-		return nil
+		return err
```

Comment:
Remove the above warning log and return `errors.Wrapf(err, "failed to detect the filesystem for volume %v before the remount", volumeName)`

Reply:
And actually this error return will affect the following case. You need to consider and check it. And it's better to add the case in the manual test plan:
A volume without the filesystem crashes unexpectedly. Longhorn needs to reattach it but skip the remount.

Reply:
In --A volume without the filesystem crashes unexpectedly. Longhorn needs to reattach it but skip the remount.--, do you mean that the volume is attached to a node but are not mounted to any pod?

Reply:
Oh, the caller (volume-controller sync loop) won't be interrupted when this function errors out. Then it's safe to error here or below.

And one more step for the manual test:
Verify the volume can be detached.